populate/src/main/java/bio/terra/pearl/populate/service/KitTypePopulator.java

```diff
@@ -0,0 +1,47 @@
+package bio.terra.pearl.populate.service;
+
+import bio.terra.pearl.core.dao.kit.KitTypeDao;
+import bio.terra.pearl.core.model.kit.KitType;
+import bio.terra.pearl.populate.service.contexts.FilePopulateContext;
+import org.springframework.stereotype.Service;
+
+import java.io.IOException;
+import java.util.Optional;
+
```

Comment:
I ended up not needing a separate `KitRequestPopulator` because it's rolled into `EnrolleePopulator` to avoid duplicating profile/address information (although it is technically possible for a kit request to have a different address than the current profile address). Although now that I think about it, the current enrollee could be passed to `KitRequesntPopulator` through a custom `FilePopulateContext`... but I'm inclined to say that this works well enough for now.